src/main/java/com/google/sps/servlets/PostAnswerServlet.java

```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** 
+ * This servlet will post an answer to a question.
+ */
+@WebServlet("/post-answer")
+public class PostAnswerServlet extends HttpServlet {
+
+  /** 
+   * This method will execute the query to insert an answer to the database.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String body = request.getParameter("answer-body");
+    int question_id = Integer.parseInt(request.getParameter("question_id"));
+    int author_id = Utility.getUserId();
+
+    try {
+      Connection connection = DriverManager.getConnection(
+        Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+      insertNewAnswer(connection, question_id, body, author_id);
+      insertNewFollower(connection, author_id);
+    } 
+    catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(PostAnswerServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
+    response.sendRedirect("/question.html?id=" + question_id);
+  }
```

Comment:
Up to debate, but you could maybe simplify it like this:
```suggestion
    insertNewAnswer(question_id, body, author_id);
    insertNewFollower(author_id);
```
This would require for you to establish the connection in each function, though, which I don't really know if it's bad.

Reply:
i prefer having just one connection, since having more might be troublesome when a lot of people are connecting, and also if there is a connection error it would be logged twice instead of just once from this try-catch

Reply:
Ok, sounds good! Then maybe I should implement this later in my files as well, to be consistent.